addons/web/static/src/views/form/form_controller.js

```diff
@@ -288,15 +290,32 @@ export class FormController extends Component {
      * is invalid or if a server error is thrown.
      * @param {Record} record
      */
-    async onRecordSaved(record) {}
+    async onRecordSaved(record) {
+        if (this.duplicateId === record.id && Object.keys(this.translationChanges).length) {
+            this.duplicateId = undefined;
```

Comment:
i'm wondering if this makes sense, because it means you have exactly *one chance* for this to run. If you click on another field that triggers an immediate save (e.g. status bar, boolean toogle, etc.), then your string change will be restricted to your language.

Removing this condition seems fine to me, as it will only survive as long as you're in a form view and will only have an impact as long as you're on the record that got duplicated. You could also clean it up when changing record if you want (e.g. when using the pager).

But i don't think we should have this work *only once*.